### ISSUE TITLE
Interactive visual debugger

### ISSUE THREAD

**Issue by AlbertoCenzato** [AlbertoCenzato/pytorch_ext#4]
It wold be interesting to have a callback that runs in "debug" mode and lets the user access training info such as:
- [ ] gradients
- [x] activations
- [ ] weights
- [ ] etc. 
from visdom_board with the possibility to stop the training to inspect data with ease.
